### DISCUSSION TITLE
Punctuation for French

### DISCUSSION THREAD

**Discussion by netw0rkf10w** [deepgram/community#801] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - STT Streaming

### Details

Hello,

I've tried Deepgram to transcribe French speech but it doesn't seem to output punctuation at all, even if I have `"punctuate": True` in the config. Could you please tell me if this is expected?

Thank you very much in advance for your reply!

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by SandraRodgers** [comment]
Hi @netw0rkf10w,

I just tested transcribing french using our [playground](https://playground.deepgram.com/?endpoint=listen) and punctuation worked for me. Are you using one of the SDKs? Could you tell me which language? And then if possible share the way you added the punctuate option to your request? Thanks!

**Reply by netw0rkf10w** [reply]
Hi @SandraRodgers. Thanks for your reply. I have tried the playground but punctuation did not work. Please see the attached screenshot. The request ID is `c0ebc34f-2f6f-4606-8a55-503f571e4cad`. Could you please check?

<img width="1324" alt="Screenshot 2024-06-07 at 19 25 16" src="https://github.com/deepgram/community/assets/8569515/54b269b0-42d7-44f6-a8be-28b974f07d5c">

**Reply by netw0rkf10w** [reply]
And the language is **French**, as specified in my original post. By the way, could you please tell me what would be the best configuration for the French language (in terms of accuracy)? Because I have tried your [beta  Enhanced feature](https://deepgram.com/learn/enhanced-french) but it only slightly improved over `nova-2`. Thank you in advance!

**Comment by SandraRodgers** [comment]
Hi @netw0rkf10w , I am looking into this.

In the meantime could you tell me if this problem happens for other audio files? Do you always have this problem of no punctuation? Could you test it with this audio in the playground? https://www.youtube.com/watch?v=FiV17kwYc_0&ab_channel=RFI

By the way, you don't need to use `punctuate` if you are using `smart_format`. Our feature `smart_format` will add punctuation and other types of formatting. So I recommend turning off punctuation if you use smart_format (but even if you turn on both, you shouldn't have this problem).

If you want to share the audio example here, that would be helpful but it's ok if you don't want to do that. I will check if I can find any information based on your request ID.

Sandra

**Reply by netw0rkf10w** [reply]
Hi @SandraRodgers. It doesn't happen all the time, but most of the time. It seems the support for French is still very limited... 
The test audio file is attached for your convenience (please change the extension to *.m4a).

[Test Deepgram for French.txt](https://github.com/user-attachments/files/15775900/Test.Deepgram.for.French.txt)

**Reply by SandraRodgers** [reply, marked as answer]
Thanks, I'm seeing the same thing with that audio you sent. I'll see what I can find out and get back to you in the next couple days. Thank you for your patience.

**Reply by netw0rkf10w** [reply]
Hi @jpvajda. Could you please tell me why you closed this issue? I've tried again and it seems the problem still persists. Thanks.
